modules/Vtiger/models/Record.php

```diff
@@ -1637,7 +1637,31 @@ public function getRecordRelatedListViewLinksLeftSide(Vtiger_RelationListView_Mo
 				]);
 			}
 		}
-		return $links;
+		return $this->returnSequenceLinks(App\Config::module($this->getModuleName(), 'relatedListViewButtonSequence', []), $links);
+	}
+
+	/**
+	 * function return links for record List and related List in the specified sequence.
+	 *
+	 * @param array $config
+	 * @param array $links
+	 *
+	 * @return array
+	 */
+	public function returnSequenceLinks ($config, $links) {
```

Comment:
rename functon or delete
add type for arguments
improve DocBlock description
optimize your operations